utils/linkage.py

```diff
@@ -135,6 +270,77 @@ def get_street_from_address_line_1(address_line_1: str) -> str:
     return " ".join(string)
 
 
+def convert_duplicates_to_dict(df: pd.DataFrame) -> pd.DataFrame:
+    """Saves to the "output" directory a file mapping multiple strings to one
+    string
+
+    Given a dataframe where each row contains one string in a column and a list
```

Comment:
While I think I understand what this function does, I don't think this explanation articulates it...